src/main/java/org/broadinstitute/hellbender/tools/copynumber/utils/annotatedregion/SimpleAnnotatedGenomicRegion.java

```diff
@@ -85,23 +110,29 @@ public String toString() {
     }
 
     /**
-     *  Reads entire TSV file in one command and stores in RAM.  Please see {@link SimpleAnnotatedGenomicRegion::CONTIG_HEADER},
-     *  {@link SimpleAnnotatedGenomicRegion::START_HEADER}, and
-     *  {@link SimpleAnnotatedGenomicRegion::END_HEADER} for defining the genomic region.
+     *
+     * This method will likely be removed in future releases.
+     *
+     *  See {@link SimpleAnnotatedGenomicRegion#readAnnotatedRegions(File, Set)}, except that all columns (except those to define the interval)
+     *   are considered headers of interest.
      *
      * @param tsvRegionFile -- File containing tsv of genomic regions and annotations per line.  E.g. a segment file.
-     * @param headersOfInterest -- should not include any headers that are used to define the region (e.g. contig, start, end)
-     * @return annotated regions with one line in the input file for each entry of the list.  Never {@code null}
+     * @return annotated regions with one line in the input file for each entry of the list.  Never {<@UNVERIFIED|@code> null}
      */
-    public static List<SimpleAnnotatedGenomicRegion> readAnnotatedRegions(final File tsvRegionFile, final Set<String> headersOfInterest) {
+    public static List<SimpleAnnotatedGenomicRegion> readAnnotatedRegions(final File tsvRegionFile) {
```

Comment:
Also vetoing this.  Put sequence dictionaries in your files if you want to obey the CNV collections format.

Reply:
It's now required everywhere.

Done.